balloon-hash/src/error.rs

```diff
@@ -45,6 +48,7 @@ impl From<Error> for password_hash::Error {
             Error::ThreadsTooFew => InvalidValue::TooShort.param_error(),
             Error::ThreadsTooMany => InvalidValue::TooLong.param_error(),
             Error::TimeTooSmall => InvalidValue::TooShort.param_error(),
+            Error::OutputSize => password_hash::Error::OutputTooShort,
```

Comment:
This isn't exactly correct, but I hope it's alright? Alternatively we add a new error variant to `password_hash::Error`?

Reply:
Can you open an issue on https://github.com/rustcrypto/traits ?

Reply:
I just opened a PR instead https://github.com/RustCrypto/traits/pull/1026, hope that's okay.

Reply:
Sure, except it seems like if you changed `balloon_hash::Error::OutputSize` to track an expected/actual value similar to what you propose in https://github.com/RustCrypto/traits/pull/1026 it should be possible to conditionally emit `password_hash::Error::OutputTooShort`/`OutputTooLong` correctly

Reply:
Yeah, that's a good idea.

Reply:
Done.